maintainers/responsibilities.md

```diff
@@ -0,0 +1,48 @@
+# Maintainer responsibilities
+
+## Notifications
+
+A maintainer must watch discussions and proposals from the different
+communication channels: GitHub, Discourse, and Matrix.
+
+To monitor GitHub, you must have your own entry in the `.github/CODEOWNERS`
```

Comment:
Problem: `CODEOWNERS` requires merge access, and we require `CODEOWNERS` entries to notify people (it seems).

@edolstra There are different permission levels on GitHub, maybe we can have a different permission for maintainers, e.g. "triage".

@yukiisbored no:

> The people you choose as code owners must have write permissions for the repository.  ([GitHub docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners))

@fricklerhandwerk is it okay for you @edolstra that more people get merge access to take care of the manual?

@edolstra there should be a different way such as in Nixpkgs to notify maintainers without requiring merge access

@infinisil what if some people on the Nix team take responsibility of documentation and are the only ones to merge PRs.

@yukiisbored https://github.com/palantir/policy-bot but this is much more complicated than `CODEOWNERS`

@infinisil GitHub marketplace has tons of extra bots we may be able to just plug in

@Mic92 https://github.com/marketplace/actions/auto-assign-reviewer-by-files this should be plug-and-play - will smash this in the Nix repo

Reply:
Nix documentation team agrees: documentation team members should receive merge access to the repositories they take care of. If [alternative options](https://github.com/NixOS/nix/pull/7277) pan out, we can incorporate these later. For now, assume `CODEOWNERS` and merge access.